api/views/role_requests_views.py

```diff
@@ -0,0 +1,22 @@
+from flask import Blueprint
+
+from api.extensions import Api, docs
+from api.views.resources import RoleRequestList, RoleRequestResource
+
+bp_name = "api-role-requests"
+bp_url_prefix = "/api/role_requests"
```

Comment:
We should use hyphens instead of underscores for URL paths

```suggestion
bp_url_prefix = "/api/role-requests"
```